tests/unit/test_map.py

```diff
@@ -75,3 +75,46 @@ def test_map_empty_geometry_column():
         "entry-date,geometry,legislation,name,notes,organisation,point,prefix,"
         "reference,start-date\r\n,,,,,,MULTIPOLYGON(),,,,,,,,\r\n"
     )
+
+
```

Comment:
You shouldn't be working on multiple bits of work in one PR. the mapping PR shouldn't really be touching any of the same code. Also this is testing the map phase so should be in tests/unit/phase/map.py